cg/store/models.py

```diff
@@ -515,7 +515,7 @@ def to_dict(self, links: bool = False, analyses: bool = False) -> dict:
         return data
 
 
-class FamilySample(Model):
+class CaseSample(Model):
     __tablename__ = "family_sample"
     __table_args__ = (UniqueConstraint("family_id", "sample_id", name="_family_sample_uc"),)
```

Comment:
```suggestion
    __table_args__ = (UniqueConstraint("case_id", "sample_id", name="_case_sample_uc"),)
```

Reply:
Not remaining the table stuff yet, my PR description was a bit off